src/components/Checkbox/Checkbox.types.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable max-len */
+
+/**
+ * @deprecated this prop is no longer necessary as this component should always be used with the primary color.
+ */
+export type CheckboxColors = 'primary' | 'secondary';
+
+export interface CheckboxProps {
+  /**
```

Comment:
acho que aqui eu inverteria a ordem das informações. Avisaria primeiro que é uma propriedade que foi descontinuada e depois explicaria o que ela faz. O que acha?

Reply:
De acordo 👍

Reply:
Achei legal sua sugestão, fiz isso e adicionei uma cor vermelha e itálico a mensagem. Acho que ficou mais nítido e fácil de ver sobre o atributo.